components/window/overview.md

```diff
@@ -172,6 +171,7 @@ In Blazor, however, the render tree structure may be important. In some cases, t
 * [Configure the Window position](slug:components/window/position)
 * [Set the Window size, minimized, and maximized state](slug:components/window/size)
 * [Handle Window events](slug:window-events)
+* [Learn How to Programmatically Center the Window](slug:window-kb-center-programmatically)
```

Comment:
The reader should get (and expect) to learn this from the Position article 3 lines above. We can link this KB only from the Position article.